pkg/gofr/datasource/opentsdb/opentsdb_test.go

```diff
@@ -182,7 +184,7 @@ func TestPutInvalidDataPoint(t *testing.T) {
 
 	err := client.PutDataPoints(context.Background(), dataPoints, "", resp)
 	require.Error(t, err)
-	require.Equal(t, "the value of the given datapoint is invalid", err.Error())
+	require.Equal(t, "invalid data points: please give a valid value", err.Error())
```

Comment:
```suggestion
	require.EqualError(t, err, "invalid data points: please give a valid value")
```

But don't you have a sentinel so you could use ErrorIs?